src/main/java/org/stellar/sdk/xdr/ClaimableBalanceID.java

```diff
@@ -4,18 +4,16 @@
 package org.stellar.sdk.xdr;
 
 
-import java.io.IOException;
-
 import com.google.common.base.Objects;
 
+import java.io.IOException;
+
 // === xdr source ============================================================
 
 //  union ClaimableBalanceID switch (ClaimableBalanceIDType type)
 //  {
 //  case CLAIMABLE_BALANCE_ID_TYPE_V0:
 //      Hash v0;
-//  case CLAIMABLE_BALANCE_ID_TYPE_FROM_POOL_REVOKE:
-//      Hash fromPoolRevoke;
```

Comment:
unrelated? :thinking: I don't remember seeing this in the Go XDR update

Reply:
I think so, I diff'd *.x from p19 branch and here and there was some slight changes, I recopied all the .x from p19 branch into the java sdk, re-ran the xdrgen on those. cleaned this up.